filament/backend/include/private/backend/MetalPlatform.h

```diff
@@ -20,7 +20,7 @@
 #include <backend/DriverEnums.h>
 #include <backend/Platform.h>
 
-#import <Metal/Metal.h>
+#include <Metal/Metal.h>
```

Comment:
#import is valid on macOS here, but #include is fine too.

Reply:
I don't understand what this change has to do with the rest of the PR or why it's necessary. I'd prefer to keep the `#import` unless there's a specific reason.